spec/controllers/hub/clients_controller_spec.rb

```diff
@@ -804,91 +811,27 @@
         end
       end
 
-      context "when the statuses are the same as the current statuses" do
-        let(:new_status_2019) { "intake_in_progress" }
-        let(:new_status_2018) { "intake_in_progress" }
-
-        it "does not create a system status change note" do
-          expect do
-            post :update_take_action, params: params
-          end.not_to change(SystemNote, :count)
-        end
-      end
-
       context "creating a note" do
         let(:internal_note_body) { "Lorem ipsum note about client tax return" }
+        let(:new_status_2019) { "intake_more_info" }
 
         it "saves a note" do
           expect do
             post :update_take_action, params: params
           end.to change(Note, :count).by(1)
 
-          note = Note.last
-          expect(note.client).to eq client
-          expect(note.body).to eq internal_note_body
-          expect(note.user).to eq user
-
-          expect(flash[:notice]).to match "Added internal note"
-        end
-      end
-
-      context "when the note field is blank" do
-        let(:internal_note_body) { " \n" }
-
-        it "does not save a note" do
-          expect do
-            post :update_take_action, params: params
-          end.not_to change(Note, :count)
-        end
-      end
-
-      context "when the message body is present" do
-        let(:message_body) { "There's always money in the banana stand" }
-
-        context "and the contact method is email" do
-          let(:contact_method) { "email" }
-          let(:locale) { "es" }
-          before { allow(subject).to receive(:send_email) }
-
-          it "sends an email using the form locale and mentions that in the flash message" do
-            post :update_take_action, params: params
-
-            expect(subject).to have_received(:send_email).with(
-              "There's always money in the banana stand", subject_locale: "es"
-            )
-            expect(flash[:notice]).to match "Sent email"
-          end
-        end
-
-        context "and the contact method is text message" do
-          let(:contact_method) { "text_message" }
-          before { allow(subject).to receive(:send_text_message) }
-
-          it "sends a text message and adds that to the flash message" do
-            post :update_take_action, params: params
-
-            expect(subject).to have_received(:send_text_message).with("There's always money in the banana stand")
-            expect(flash[:notice]).to match "Sent text message"
-          end
-        end
-      end
-
-      context "when the message body is blank" do
-        let(:message_body) { " \n" }
-        let(:contact_method) { "email" }
-
-        it "does not send a message using the chosen contact method" do
-          expect do
-            post :update_take_action, params: params
-          end.not_to change(OutgoingEmail, :count)
+          expect(flash[:notice]).to include "added internal note"
```

Comment:
Consider mocking the `TakeActionForm` object, or some state on it, in these controller specs. As I understand it, "added internal note" is text that comes from `TakeActionForm`, so it might be advantageous to avoid testing it here.